client/src/pages/Login/useStyles.ts

```diff
@@ -2,33 +2,28 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles(() => ({
   root: {
-    minHeight: '100vh',
-    width: '100%',
+    minHeight: '60vh',
+    marginTop: '6rem',
+    marginBottom: '3rem',
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
```

Comment:
Could you elaborate on this a bit more? I don't see where this is being effective. Looking at your application I can't find this border being applied after changing it to a colour that would be immediately noticeable. As far as I see, this is unnecessary to have and can be removed.

Reply:
Yes, you are right. This is unecessary. I will remove it.